Iceberg-Plugin-GitHub/IceGitHubAcceptPullRequestPresenter.class.st

```diff
@@ -16,6 +16,11 @@ Class {
 	#tag : 'View'
 }
 
+{ #category : 'accessing' }
+IceGitHubAcceptPullRequestModel class >> defaultPreferredExtent [
+	^ (600 @ 300)
+]
+
 { #category : 'class initialization' }
 IceGitHubAcceptPullRequestPresenter class >> initialize [
```

Comment:
Hi guys, this PR is breaking the Pharo build.

I don't know how did you arrive to this, but look, the same file has methods with different class names.

Reply:
That was my fault, I fixed a merge conflict but I did not notice those name changes. 

I redid a PR fixing those errors